### DISCUSSION TITLE
Different color when disabled

### DISCUSSION THREAD

**Discussion by eduardodallmann** [tailwindlabs/tailwindcss#11792] [Help] [answered]
Hello, I can't understand why the input and select background colors are different when disabled.

Am I doing something wrong?

[codesandbox](https://codesandbox.io/s/funny-payne-k8z7pk?file=/src/App.js)

**Comment by wongjn** [comment, marked as answer]
At least in Chrome, disabled `<select>` elements have a default `opacity` of `0.7`, whereas disabled `<input>` elements have an unchanged opacity of `1`.

![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/42ff5af2-337f-49fe-aeae-3705d2f343b5)

Consider explicitly applying `opacity: 1` to the `<select>` element via `opacity-100` or applying `opacity: 0.7` to the `<input>` via `opacity-70`: https://play.tailwindcss.com/FrKf3CXEjd

**Reply by OsalexRod** [reply]
gracias, me ayudo a solucionar el mismo problema.

**Reply by ryanmortier** [reply]
I struggled with this for a while today. Such an insane thing for a browser to do. Thanks for your post.
